core/src/main/java/nl/nn/adapterframework/util/Locker.java

```diff
@@ -263,6 +265,26 @@ public void release(String objectIdWithSuffix) throws JdbcException, SQLExceptio
 		}
 	}
 
+	public String getLockerInfo(String objectIdWithSuffix) {
+		try {
+			String query = getDbmsSupport().prepareQueryTextForNonLockingRead(selectQuery);
+			try (Connection conn = getConnection(); PreparedStatement stmt = conn.prepareStatement(query)) {
+				stmt.clearParameters();
+				stmt.setString(1,objectIdWithSuffix);
+				
+				try (ResultSet rs = stmt.executeQuery()) {
+					if (rs.next()) {
+						String info = "objectId ["+objectId+"] current type ["+rs.getString(1)+"] current host ["+rs.getString(2)+"] current creationDate ["+DateUtils.format(rs.getTimestamp(3))+"] current expirydate ["+DateUtils.format(rs.getTimestamp(4))+"]";
```

Comment:
Ik denk dat 'current' een beetje overbodig is.

```suggestion
						String info = "objectId ["+objectId+"] of type ["+rs.getString(1)+"]. Process locked by host ["+rs.getString(2)+"] at ["+DateUtils.format(rs.getTimestamp(3))+"] with expiry date ["+DateUtils.format(rs.getTimestamp(4))+"]";
```